manifests/g/Getscreenme/Dashboard/1.0.16/Getscreenme.Dashboard.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Getscreenme.Dashboard
+PackageVersion: 1.0.16
+PackageLocale: en-US
+Publisher: Getscreen.me
+PublisherUrl: https://getscreen.me
+PublisherSupportUrl: https://getscreen.me/en/resources
+PrivacyUrl: https://getscreen.me/en/privacy-policy
+Author: Point B Ltd
+PackageName: Getscreen.me Dashboard
+PackageUrl: https://getscreen.me/en/download-dashboard/
+License: Proprietary
+LicenseUrl: https://getscreen.me/en/terms-of-use/
+Copyright: © 2018-2022 Getscreen.me
+ShortDescription: Dashboard is a desktop application that duplicates the functions of a personal account and adds comfort for lengthy connections
+Moniker: Getscreenme.Dashboard
+PurchaseUrl: https://getscreen.me/en/plan
+Tags:
+- Remote control
+- Dashboard
+- Getscreen.me
```

```suggestion
- remote-control
- dashboard
- getscreen.me
```
Tags should be written in all lowercase with hypens used instead of spaces.